WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1487,6 +1487,16 @@ private extension ProductFormViewController {
     }
 }
 
+private extension ProductFormViewController {
+    func productOrVariationID() -> ProductOrVariationID {
```

Comment:
I would prefer to use a computed property for added clarity at the calling place. A function call makes me check twice about what other things the function does. 😄